packages/bunnymen-core/src/node.ts

```diff
@@ -36,6 +36,7 @@ export interface INodeEvents {
 }
 
 export class Node extends EventEmitter {
+  private BASE_TOPIC = '_peer-discovery._p2p._pubsub'
```

Comment:
What is the scope of this namespace? Our protocol?

Reply:
this is just the "recommended" format